src/test/resources/tf/GCP-GKE-Client-Certificate-is-set-to-Disabled/GCP-GKE-Client-Certificate-is-set-to-Disabled-negative.json

```diff
@@ -0,0 +1,81 @@
+{
+  "resource": [
+    {
+      "google_container_cluster": [
+        {
+          "primary": [
+            {
+              "initial_node_count": 1,
+              "location": "us-central1",
+              "master_auth": [
+                {
+                  "client_certificate_config": [
+                    {
+                    }
+                  ],
+                  "password": "",
+                  "username": ""
+                }
+              ],
+              "name": "my-gke-cluster",
+              "remove_default_node_pool": true
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "google_container_cluster": [
+        {
+          "secondary": [
+            {
+              "initial_node_count": 1,
+              "location": "us-central1",
+              "master_auth": [
+                {
+                  "client_certificate_config": [
+                    {
```

Comment:
By default, the value for  "issue_client_certificate" parameter is false. Therefore omitting it will make it a positive test case. In this case the rule should match. Pls update your rule accordingly.

Reply:
@ngarg-panw seems false is good :)